### PR TITLE
Updates autocomplete component to match redesign

### PR DESCRIPTION
**Pull request by mmrtnz**
[AV-3226](https://lobsters.atlassian.net/browse/AV-3226) - Redesign autocomplete to match react autocomplete

vue

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
 <div>
-  <AddressAutocomplete apiKey="YOUR_API_KEY_HERE" :addresses="addresses" @selectItem="selectItem" @newSuggestions="addNewSuggestions" />
+  <AddressAutocomplete apiKey="live_pub_16d30adbb46a1c360ebf7d1e6b48361" :addresses="addresses" @selectItem="selectItem" @newSuggestions="addNewSuggestions" />
```
Comment (Thexumaker) [inline src/App.vue:3]: small nit but replace this with your api key here

**Review by Thexumaker — COMMENTED** [review]
LGTM except 1 quick question, are we planning on formatting the input text box itself as well?

Also I can handle fixing up some of the readme's and getting this published but we should also create a wiki like the react component :3

File: src/App.vue (hunk 1 above)
Comment (mmrtnz) [reply src/App.vue:3]: nice catch! 🙏

**Comment by mmrtnz** [comment]
@Thexumaker Yeah we'll update the input to match react's

**Pull request merged**